client/themes/morning.css

```diff
@@ -114,31 +112,36 @@ QUIT #d0907d
   border-color: #242a33;
 }
 
-#form .input {
-  margin-right: 0;
-}
-
 #form #input {
   background-color: #2e3642;
   border-color: #242a33;
   color: #cccccc;
-  padding-left: 1em !important;
 }
 
 #form #nick {
-  display: none;
+  background: #242a33;
+  color: #CCC;
+}
+
+/* Buttons */
+#chat .show-more-button,
+#form #submit,
+#windows .header .button {
+  background: #2e3642;
+  border-color: #242a33;
+  color: #CCC;
+}
+
+#chat .show-more-button:hover,
+#form #submit:hover,
+#windows .header .button:hover {
+  color: #FFF;
 }
 
 
 #chat .header {
   color: #99a2b4;
 }
-/* Hide unnecessary buttons */
-#windows .header .button,
-#form #submit {
-  display: none;
```

Comment:
Yeah, I agree with this. It's confusing if a user changes a theme and then can't leave any rooms on mobile. took me ages to figure out this is why there wasn't a leave button.